README.md

```diff
@@ -1,88 +1,88 @@
-FireMarshal
-==================================
+# FireMarshal
 
-This tool builds base images for several linux-based distros that work with qemu,
-spike, and firesim.
+FireMarshal is a workload generation tool for RISC-V based systems that automates constructing boot binaries and filesystem images and their evaluation.
 
-This is just a quick primer. To see full documentation, please see the official
-documentation:
-https://firemarshal.readthedocs.io/en/latest/index.html
+## Quick Links
 
-# Requirements
-The easiest way to use Marshal is to run it via Chipyard
-(https://chipyard.readthedocs.io/en/latest/) or FireSim
-(https://docs.fires.im/en/latest/). However, this is not required. To run
-FireMarshal independently, you will need the following dependencies:
+* **Stable Documentation**: https://firemarshal.readthedocs.io/en/latest/index.html
+* **Paper Publication**: https://ieeexplore.ieee.org/document/9408192
+* **Bugs and Feature Requests**: https://github.com/firesim/FireMarshal/issues
+* **General Questions, Help, Discussion**: https://groups.google.com/forum/#!forum/firesim
+* **Want to contribute?**: https://github.com/firesim/FireMarshal/blob/master/CONTRIBUTING.md
 
-## Standard Packages
-``centos-requirements.txt`` is a list of packages for centos7 that are needed by
-marshal. You can install these with:
-```
-cat centos-requirements.txt | sudo xargs yum install -y
-```
+## Setup
 
-``ubuntu-requirements.txt`` is a list of packages for Ubuntu 18.04 that are needed by marshal.
-You can install these with:
-```
-cat ubuntu-requirements.txt | sudo xargs apt-get install -y
-```
+### ::RECOMMENDED:: Chipyard/FireSim Integration
 
-Package names may be different on other distributions.
+The easiest way to use FireMarshal is to run it via [Chipyard](https://chipyard.readthedocs.io/en/latest/) or [FireSim](https://docs.fires.im/en/latest/).
+However, this is not required.
+To run FireMarshal independently, follow along from (#standalone-setup).
```

Comment:
I don't think this creates a link in Github markdown. (IMO a link isn't necessary here
```suggestion
To run FireMarshal independently, follow along from the next section (Standalone setup).
```